README.md

```diff
@@ -70,30 +70,18 @@ Getting Started
 
 ### Installation
 
-The best way to install pynapple is with pip within a new [conda](https://docs.conda.io/en/latest/) environment :
-
+The best way to install pynapple is with pip inside a new [conda](https://docs.conda.io/en/latest/) environment:
     
 ``` {.sourceCode .shell}
 $ conda create --name pynapple pip python=3.8
 $ conda activate pynapple
 $ pip install pynapple
 ```
 
-or directly from the source code:
-
-``` {.sourceCode .shell}
-$ conda create --name pynapple pip python=3.8
-$ conda activate pynapple
-$ # clone the repository
-$ git clone https://github.com/pynapple-org/pynapple.git
-$ cd pynapple
-$ # Install in editable mode with `-e` or, equivalently, `--editable`
-$ pip install -e .
```

Comment:
Note: Moved this to the contributor guide because I don't think you'd usually install from source unless you're editing the library - but can move back if that's incorrect